dashboard/test/fixtures/video.yml

```diff
@@ -0,0 +1,12 @@
+video_13:
+  id: 13
+  key: hoc_wrapup
+  youtube_code: 98Wft30gUQE
+  download: http://videos.code.org/new/2f-HoC-wrap-up-final.mp4
+  locale: en-US
+video_137:
+  id: 137
+  key: csd_gamelab_draw_2
+  youtube_code: QLabeWhw_O0
+  download: https://videos.code.org/levelbuilder/gamelab_addingcolors-mp4.mp4
+  locale: en-US
```

Comment:
**Why these two fixtures?** For some reason [13]() and [137]() were the only video ids I found in any of our existing fixtures, so I pulled rows 13 and 137 from videos.csv and used them here.